kubeflow-pipelines-scheduledworkflow.yaml

```diff
@@ -0,0 +1,64 @@
+package:
+  name: kubeflow-pipelines-scheduledworkflow
```

Comment:
I think we can put all kubeflow-pipelines related packages under one main one and then have subpackages for all the pipeline components?

I.e. I had a tinker a little while back at using data ranges, something like this https://github.com/wolfi-dev/os/pull/2206